### PR TITLE
SubQueue.enable/disable operations should be idempotent

### PR DESCRIPTION
**Pull request by mbettiol**
Enabling already enabled queues or disabling already disabled queues computed a wrong MultiQueue totalSize causing NullPointer on take/poll operations

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .gradle/
 build/
+.idea/
```
Comment (marianobarrios) [inline .gitignore:3]: This change is unrelated to the main change

File: src/main/java/lbmq/LinkedBlockingMultiQueue.java [hunk 2]
```diff
@@ -511,6 +511,10 @@ public void clear() {
     public void enable(boolean status) {
       fullyLock();
       try {
+        boolean notChanged = status == enabled;
+        if(notChanged){
```
Comment (marianobarrios) [inline src/main/java/lbmq/LinkedBlockingMultiQueue.java:515]: Please respect formatting (spaces).

**Review by marianobarrios — COMMENTED** [review]
Thanks. Left two minor comments.

File: .gitignore (hunk 1 above)
Comment (mbettiol) [reply .gitignore]: removed

File: src/main/java/lbmq/LinkedBlockingMultiQueue.java (hunk 2 above)
Comment (mbettiol) [reply src/main/java/lbmq/LinkedBlockingMultiQueue.java:515]: I may be wrong but spaces seem already beeing used.
